### PR TITLE
Removed static test values for cultures which could change, e.g. Win1…

### PR DESCRIPTION
**Pull request by ViktorHofer**
Removed static test values for cultures which could change, e.g. Win10 Insider builds

fixes https://github.com/dotnet/corefx/issues/18208

### CODE REVIEW

File: src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentNegativePattern.cs [hunk 1]
```diff
@@ -12,21 +12,30 @@ public class NumberFormatInfoPercentNegativePattern
...
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
         [MemberData(nameof(PercentNegativePattern_TestData))]
-        public void PercentNegativePattern_Get(NumberFormatInfo format, int expectedWindows, int expectedIcu)
+        public void PercentNegativePattern_Get(NumberFormatInfo format, int expected)
         {
-            int expected = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? expectedWindows : expectedIcu;
             Assert.Equal(expected, format.PercentNegativePattern);
         }
 
         [Theory]
```
Comment (tarekgh) [inline src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentNegativePattern.cs]: should make this one as Fact?

File: src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentPositivePattern.cs [hunk 2]
```diff
@@ -12,21 +12,30 @@ public class NumberFormatInfoPercentPositivePattern
...
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
         [MemberData(nameof(PercentPositivePattern_TestData))]
-        public void PercentPositivePattern_Get(NumberFormatInfo format, int expectedWindows, int expectedIcu)
+        public void PercentPositivePattern_Get(NumberFormatInfo format, int expected)
         {
-            int expected = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? expectedWindows : expectedIcu;
             Assert.Equal(expected, format.PercentPositivePattern);
         }
 
         [Theory]
```
Comment (tarekgh) [inline src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentPositivePattern.cs]: ditto. make it as Fact

File: src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentNegativePattern.cs [hunk 3]
```diff
@@ -12,21 +12,30 @@ public class NumberFormatInfoPercentNegativePattern
...
-            yield return new object[] { new CultureInfo("en-MY").NumberFormat, 1, 1 };
-            yield return new object[] { new CultureInfo("tr").NumberFormat, 2, 2 };
+            yield return new object[] { new CultureInfo("en-US").NumberFormat, 1 };
+            yield return new object[] { new CultureInfo("en-MY").NumberFormat, 1 };
+            yield return new object[] { new CultureInfo("tr").NumberFormat, 2 };
         }
 
+        /// <summary>
+        /// Not testing for Windows as the culture data can change
+        /// https://blogs.msdn.microsoft.com/shawnste/2005/04/05/culture-data-shouldnt-be-considered-stable-except-for-invariant/
```
Comment (tarekgh) [inline src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentNegativePattern.cs]: I would suggest changing/add the comment telling how we are testing the same thing for Windows in the CultureAllTest.

**Comment by ViktorHofer** [comment]
@tarekgh: thanks, improved comments and changed to fact.

**Comment by tarekgh** [comment]
LGTM. Thanks

**Pull request merged**